ci/debug_install_data_services.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu -o pipefail
+
+readonly HOST_DCOS_CLI_DIRECTORY="${DCOS_DIR:-${HOME}/.dcos}"
+docker run -it --rm \
+  -v "${HOST_DCOS_CLI_DIRECTORY}:/dcos-cli-directory":ro \
+  -v "$(pwd)/../python:/dcos-service-diagnostics" \
+  -v "$(pwd):/ci" \
+  -w "/ci" \
+  -e DCOS_SERVICE_DIAGNOSTICS_DIRECTORY=/dcos-service-diagnostics \
+  -e DCOS_CLI_AUTO_INIT="yes" \
+  "mesosphere/dcos-sdk-service-diagnostics:v0.4.0" \
```

Comment:
I'm somewhat afraid that we will forget to change this and will keep testing the wrong version :-/

Reply:
Me too, but once we do parametrize `create_service_diagnostics_bundle.sh` script (o'm creating separate task for that) the concerns should get rid.

Reply:
Ah, OK, I forgot we agreed to do it separately :man_facepalming: .

Reply:
the comments added